src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,115 @@
-export const PeopleTable = () => {
```

Comment:
You are missing the import statement for React component here. Add `import React from 'react';` at the beginning.